packages/components/src/components/primitives/Progress/Progress.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { Progress as ChakraProgress } from '@chakra-ui/react';
+import { ProgressProps } from './types';
+
+export const Progress: FC<ProgressProps> = (props) => {
+  return (
+    <ChakraProgress
+      {...props}
+      data-testid="composer-progressbar"
+    ></ChakraProgress>
+  );
+};
```

Comment:
1. Remove the curly braces and `return` since we have a single line.
2. Use dot notation for the test id instead of "composer-...." use "composer...."
3. Change this component to be a self closing tag `<ChakraProgress .../>`